.github/workflows/scala-ci.yml

```diff
@@ -6,6 +6,10 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - spark-version: 3.4.0
+            scala-version: 2.13.10
+          - spark-version: 3.4.0
+            scala-version: 2.12.12
           - spark-version: 3.3.2
             scala-version: 2.13.10
```

Comment:
other scala versions are also not accurate

Reply:
I tried this and then I relearned why this was pegged to 2.12.12, the later versions are not compatible with the version of sbt-scoverage that is used and later versions require sbt >= 1.0 and to address that one need to drop the usage/upgrade of `sbt-spark-package` to support sbt >= 1.0. This should probably be done but it seems big enough that it better done in other PRs.

Reply:
so annoying...